frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -447,142 +465,226 @@ export function Experiment(): JSX.Element {
+                                </div>
+                            </Row>
+                            <CodeSnippet language={Language.JavaScript}>
+                                {`posthog.feature_flags.override({'${experimentData.feature_flag_key}': '${currentVariant}'})`}
+                            </CodeSnippet>
+                            <CodeSnippet language={Language.JavaScript} wrap>
+                                {`if (posthog.getFeatureFlag('${
+                                    experimentData.feature_flag_key ?? ''
+                                }') === '${currentVariant}') {
+    // where '${currentVariant}' is the variant, run your code here
+}`}
+                            </CodeSnippet>
+                            <a
+                                target="_blank"
+                                rel="noopener noreferrer"
+                                href="https://posthog.com/docs/user-guides/feature-flags"
+                            >
+                                <Row align="middle">
+                                    Experiment implementation guide
+                                    <IconOpenInNew className="ml-05" />
+                                </Row>
+                            </a>
                         </Col>
                     </Row>
-                    {experimentData.start_date && (
-                        <div className="experiment-result">
-                            {experimentData.end_date && <Alert type="info" message="This experiment has ended" />}
-                            <div>Experiment start date: {dayjs(experimentData.start_date).format('D MMM YYYY')}</div>
-
-                            {experimentResults && (
-                                <BindLogic
-                                    logic={insightLogic}
-                                    props={{
-                                        dashboardItemId: experimentResults.itemID,
-                                        filters: {
-                                            ...experimentResults.filters,
-                                            insight: 'FUNNELS',
-                                            funnel_viz_type: FunnelVizType.Steps,
-                                            display: 'FunnelViz',
-                                            layout: FunnelLayout.vertical,
-                                        },
-                                        cachedResults: experimentResults.insight,
-                                        syncWithUrl: false,
-                                        doNotLoad: true,
-                                    }}
-                                >
+                    <div className="experiment-result">
+                        {experimentResults && !experimentResults.noData ? (
+                            <Row style={{ alignItems: 'baseline' }}>
+                                {experimentData.end_date ? (
```

Comment:
This should instead be a threshold (say, 2 days from start of experiment).

We want to show the probabilities anytime after that!

Reply:
(ok to do this later on!)

Reply:
yep we could think about this more along with the loading state of results that take a while to come back